lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -312,7 +287,8 @@ def self.is_hotfix?(version)
 
         # Prints the current and next release version names to stdout, then returns the next release version
         #
-        # @param [String] app The name of the app to be used for beta and alpha version update
+        # @param [String] app (UNUSED/DEPRECATED) The name of the app to be used for beta and alpha version update
+        #                     Unused anymore, will be removed in future breaking update
```

Comment:
```suggestion
        #                     No longer used, will be removed in future breaking update
```

Reply:
Fixed in 7d289b0d3972917dd81886c608d61b7f9862b2d5